### PR TITLE
Feature/set anim speed

### PR DESCRIPTION
**Pull request by Dartemiss**
What's new?
- Set animation speed from code and change value on imgui speed for more usability.
- IsGameRunning now is declared to the gameplay system.

### CODE REVIEW

File: Engine/Animation/AnimController.cpp [hunk 1]
```diff
@@ -155,6 +155,14 @@ bool AnimController::IsOnState(const std::string& state)
 	return active_state.get()->name_hash == state_hash;
 }
 
+void AnimController::SetSpeed(float speed) const
+{
+	if(active_state)
+	{
+		active_state->speed = speed;
```
Comment (yalania) [inline Engine/Animation/AnimController.cpp]: I understand this is for runtime, so in this case you should be setting the speed for the current playing animation, as this change will affect to all the models with the same state machine. playing_clips[ClipType::ACTIVE].speed is what you want

**Review by yalania — CHANGES_REQUESTED** [review]

File: Engine/Animation/AnimController.cpp (hunk 1 above)
Comment (Dartemiss) [reply Engine/Animation/AnimController.cpp]: Done!

**Review by marcorod94 — APPROVED** [review]
It looks great goo job

File: Engine/Component/ComponentAnimation.h [hunk 2]
```diff
@@ -46,6 +46,7 @@ class ComponentAnimation :	public Component
 	ENGINE_API bool IsOnState(const std::string & trigger);
 	ENGINE_API float GetCurrentClipPercentatge() const;
 	ENGINE_API int GetTotalAnimationTime() const;
+	ENGINE_API void SetSpeedAnimation(float speed) const;
```
Comment (mariofv) [inline Engine/Component/ComponentAnimation.h]: It should be `SetAnimationSpeed`

**Review by mariofv — APPROVED** [review]

**Review by JorxSS — APPROVED** [review]
GG mi pana, I left you here my refachero approval image

**Review by yalania — APPROVED** [review]

**Pull request merged**